packages/ava/__tests__/unit/data/statistics/index.test.ts

```diff
@@ -1,3 +1,7 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
What is this

Reply:
without it, I can't use "expect.extend"
![image](https://user-images.githubusercontent.com/71261480/224911465-4595b588-763e-4438-82e7-d1a1bd2b874d.png)